src/node_http_parser.cc

```diff
@@ -698,6 +704,7 @@ class Parser : public AsyncWrap {
 
 
   http_parser parser_;
+  unsigned int knownFields_[32];  // known HTTP header fields
   StringPtr fields_[32];  // header fields
   StringPtr values_[32];  // header values
```

Comment:
Consider replacing the hard-coded value with a `const int kNumFields = 32;` or something like that.